### DISCUSSION TITLE
Propose existing dashboards tags

### DISCUSSION THREAD

**Discussion by oligu** [grafana/grafana#51814] [Dashboards, Panels & Visualizations]
When entering a dashboard tag it would be useful if a list of existing tags is shown. The list would be narrowed down with every letter entered. This would prevent us from having multiple tags with only slight differences in spelling. Further it would save me from going back to the dashboard list just to search for the correct tag.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
